views/backend/write.php

```diff
@@ -8,16 +8,21 @@
 
 <?php
     if(isset($_POST['post'])){
-        $title = htmlspecialchars(trim($_POST['title']));
-        $content = htmlspecialchars(trim($_POST['content']));
+        if(isset($_POST['title'])){
+            $title = htmlspecialchars(trim($_POST['title']));
+        }
+        if(isset($_POST['content'])){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)